content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -156,8 +157,7 @@ Hit enter for command prompt
 
 / # wget --spider --timeout=1 nginx
 Connecting to nginx (10.100.0.16:80)
+remote file exists
```

Comment:
Would you be willing to tidy up this whole `console` block? At the moment it mixes commands and their output.

Reply:
Done.